template-sample-app/template.json

```diff
@@ -18,17 +18,19 @@
       "@material-ui/icons": "^4.11.2",
       "@material-ui/lab": "4.0.0-alpha.57",
       "@reduxjs/toolkit": "^1.5.0",
+      "@testing-library/cypress": "^7.0.3",
       "@testing-library/jest-dom": "^5.11.9",
       "@testing-library/react": "^11.2.3",
       "@testing-library/user-event": "^12.6.2",
       "@turf/bbox-polygon": "^6.3.0",
       "@turf/boolean-intersects": "^6.3.0",
+      "cypress": "^6.3.0",
```

Comment:
Not sure about this @aaranadev . If I remember correctly, we decided to leave cypress out, as something the client could add by himself to the project, as it involves downloading quite a bit of files (and the template initial setup was slower). Do you remember about it @borja-munoz ?

Reply:
Yes, we decided to remove it because it was adding a lot of size to the dependencies you need to download when setting up a project.

Reply:
Ok, then could you please @aaranadev remove the cypress part, please?

Reply:
Ah, of course, but since the cypress folder is published, I think you forgot. Then the cypress folder needs to be removed on publish.

Reply:
Sounds good to me. Could you please update the PR with it?